post_installation_configuration/multi-architecture-configuration.adoc

```diff
@@ -6,7 +6,12 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-An {product-title} cluster with multi-architecture compute machines is a cluster that supports compute machines with different architectures. Clusters with multi-architecture compute machines are available only on AWS or Azure installer-provisioned infrastructures and bare metal user-provisioned infrastructures with `x86_64` control plane machines. 
+An {product-title} cluster with multi-architecture compute machines is a cluster that supports compute machines with different architectures. Clusters with multi-architecture compute machines are available only on AWS or Azure installer-provisioned infrastructures and bare metal user-provisioned infrastructures with `x86_64` control plane machines.
+
+[IMPORTANT]
```

Comment:
@Prashanth684 Is this more like what you were thinking? To me it seems odd to mention this statements without giving some info or context.